ansible/roles/test/files/ptftests/advanced-reboot.py

```diff
@@ -161,6 +161,12 @@ def run(self):
             portchannel_output = "\n".join(portchannel_output.split("\r\n")[1:-1])
             sample["po_changetime"] = json.loads(portchannel_output, strict=False)['interfaces']['Port-Channel1']['lastStatusChangeTimestamp']
 
+            # Expect the routes remains during BGP graceful restart
+            po_ip_route_output = self.do_cmd("show ip route bgp | grep Port-Channel1 | wc -l")
+            po_ip_route_output = "\n".join(po_ip_route_output.split("\r\n")[1:-1])
+            po_route = int(po_ip_route_output)
+            info["po_route"] = po_route
+
```

Comment:
I believe v5_routing_ok and v6 routing ok has the route, you do not need to do another show ip route bgp.